lms/djangoapps/mobile_api/testutils.py

```diff
@@ -168,9 +169,13 @@ def test_course_not_found(self):
         response = self.api_response(expected_response_code=None, course_id=non_existent_course_id)
         self.verify_failure(response)  # allow subclasses to override verification
 
-    @skip  # TODO fix this, see MA-1038
     @patch.dict('django.conf.settings.FEATURES', {'DISABLE_START_DATES': False})
     def test_unreleased_course(self):
+        # ensure the course always starts in the future
+        # pylint: disable=attribute-defined-outside-init
+        self.course = CourseFactory.create(mobile_available=True, static_asset_path="needed_for_split")
```

Comment:
For what it's worth, I don't know why I get this pylint warning here but not in this class's subclasses.